### PR TITLE
Improve quota table sorting

### LINKED ISSUE #634: Ordering of quotas table
screen shot 2016-10-06 at 9 16 21 am

Graphs show request on left, limit on right
But table shows limits first then request

The first 4 rows of table should match the ordering of graphs (left to right => top to bottom) then additional information in rows 5+

### PR DESCRIPTION
**Pull request by spadgett**
Fixes #634

### CODE REVIEW

File: app/views/quota.html [hunk 1]
```diff
@@ -81,16 +81,16 @@ <h3 class="text-center">Memory <small>Limit</small></h3>
                         <!-- Don't show quotas for type `resourcequotas`. They are frequently at limit,
                              which is not something to worry about, and only a cluster admin can create
                              those resources anyway. -->
-                        <tr ng-repeat="(resourceType, specMax) in quota.spec.quota.hard"
+                        <tr ng-repeat="resourceType in orderedTypesByClusterQuota[quota.metadata.name]"
                             ng-if="resourceType !== 'resourcequotas'"
                             ng-class="{
                               warning: isAtLimit(quota, resourceType),
-                              disabled: (quota.status.total.hard[resourceType] || specMax) === '0'
+                              disabled: (quota.status.total.hard[resourceType] || quota.spec.total.hard[resourceType]) === '0'
```
Comment (jwforres) [inline app/views/quota.html]: if this is replacing what was being set on specMax shouldnt it be quota.spec.quota.hard ?

Comment (spadgett) [inline app/views/quota.html]: Yes, will fix

**Comment by spadgett** [comment]
@jwforres updated

**Comment by jwforres** [comment]
[merge]

**Pull request merged**